scripts/ci/ovnkube-setup.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+
+set -eu
+
+# DEPENDENCIES:
+# This scripts assumes the following packages are installed:
+#   docker
+#   go
+#   python and pip
+# PERMISSIONS:
+# This script can be executed without root privilegeds. 'sudo' will be used
+# for the specific tasks that do require it
+
+# Global env vars that modify the behavior of the script:
+KIND_VERSION=${KIND_VERSION:-v0.10.0}
+K8S_VERSION=${K8S_VERSION:-v1.20.0}
+OVNK8S_VERSION=${OVNK8S_VERSION:-ab8bcb41ae215afd3890f243fd99e1d0cce5feed}
+OVNK8S_REPO=https://github.com/ovn-org/ovn-kubernetes.git
+
+# Global vars
+WORKSPACE=${WORKSPACE:-${HOME}/ovnkube-setup}
+OS=linux
+ARCH=amd64
+export GOPATH=$WORKSPACE/go
```

Comment:
GOPATH is set by CI to $WORKSPACE

Reply:
I'm ok changing it, but this variable should not affect "upper" scripts, right? unless they are "sourced" they should run in a subshell which makes this GOPATH isolated from previous ones.

Reply:
In either case, I'll change it just to avoid having two different ones